main_test.go

```diff
@@ -116,20 +116,8 @@ func generatedFiles(t *testing.T, dir string) (map[string]string, error) {
 	return files, err
 }
 
-func TestGenerateError(t *testing.T) {
+func TestGenerateScripts(t *testing.T) {
```

Comment:
Couldn't think of a better name, but `TestGenerateError` no longer seemed relevant as this seems to be testing both success and error cases?